<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SongList</class>
 <widget class="QDialog" name="SongList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Song_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #3399FF;
}

QPushButton:hover {
    background-color: #66B2FF;
}
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Song 2: Kim larsen midt om natten</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Song_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #3399FF;
}

QPushButton:hover {
    background-color: #66B2FF;
}
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Song 1: JB - Ghost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose between these two songs:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
